models/bamf_ct_liver_tumor/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM mhubai/base:latest
+
+# FIXME: set this environment variable as a shortcut to avoid nnunet crashing the build
+# by pulling sklearn instead of scikit-learn
+# N.B. this is a known issue:
+# https://github.com/MIC-DKFZ/nnUNet/issues/1281 
+# https://github.com/MIC-DKFZ/nnUNet/pull/1209
+ENV SKLEARN_ALLOW_DEPRECATED_SKLEARN_PACKAGE_INSTALL=True
+
+# Install nnunet and platipy
+RUN pip3 install --no-cache-dir nnunetv2==2.0 \
+        dcm2niix==1.0.20220715
+
+# Install updated segdb entries
+RUN pip uninstall segdb -y
+RUN pip install git+https://github.com/MHubAI/segdb.git
```

Comment:
This can be removed; the newest base image should contain all entires now ;)

Reply:
Thanks for updating the image.